sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5663,9 +5767,27 @@ pi_result piextUSMHostAlloc(void **ResultPtr, pi_context Context,
   return PI_SUCCESS;
 }
 
-pi_result piextUSMFree(pi_context Context, void *Ptr) {
+static pi_result piextUSMFreeImpl(pi_context Context, void *Ptr) {
```

Comment:
Also name is confusing with `USMFreeImpl`. How about?
```suggestion
static pi_result USMFreeHelper(pi_context Context, void *Ptr) {
```

Reply:
Fixed.